pset-functions.js

```diff
@@ -31,6 +31,70 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+const cards = [19,20,21,22]
+
+for (i = 0; i < 10; i++) {
+
+  const playerCardScore = cards[Math.floor(Math.random() * cards.length)]
+  const dealerCardScore = cards[Math.floor(Math.random() * cards.length)]
+
+  blackJack(playerCardScore,dealerCardScore)
+
+  function blackJack(playerCardScore,dealerCardScore) {
+
+    if (playerCardScore === 22 && dealerCardScore === 22 ) {
+
+      console.log(`Player:${playerCardScore}`)
+      console.log(`Dealer:${dealerCardScore}`)
+      console.log('Both 22: 0')
+    }
+
+    else if (playerCardScore === 21 && dealerCardScore === 21 ) {
+
+      console.log(`Player: ${playerCardScore}`)
+      console.log(`Dealer: ${dealerCardScore}`)
+      console.log('Both 21: 21')
+    }
+
+    if (playerCardScore === 22 || dealerCardScore === 22 ) {
+
+      //console.log('One is over 21')
+      console.log(`Player: ${playerCardScore}`)
+      console.log(`Dealer: ${dealerCardScore}`)
+
+        if (playerCardScore === 22 &&  dealerCardScore !== 22 ) {
+
+          console.log(`Dealer wins with: ${dealerCardScore}`)
+        }
+
+        if (playerCardScore !==22 &&  dealerCardScore === 22 ) {
+
+          console.log(`Player wins with:${playerCardScore}`)
+
+        }
+      }
+
+
+  else if (playerCardScore > dealerCardScore) {
+
+    console.log(`Player:${playerCardScore}`)
+    console.log(`Dealer:${dealerCardScore}`)
+    console.log(`Player wins with:${playerCardScore}`)
+
+  }
+
+  else if (playerCardScore < dealerCardScore) {
+
+    console.log(`Player:${playerCardScore}`)
+    console.log(`Dealer:${dealerCardScore}`)
+    console.log(`Dealer wins with: ${dealerCardScore}`)
+
+  }
+    console.log(`-------------------`)
+}
+}
```

Comment:
@thetripler Great work here on the solution here. I see you created a function here to play the game and you have it looping so you can play multiple games. Good use of the array of cards values so you can get a random value to use as your parameters in your blackJack function call. 
Take a look at this solution [here]('https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L30')